src/handlers/poster/components/types/BorrelPriceListPoster.tsx

```diff
@@ -13,12 +13,12 @@ export default function BorrelPriceListPoster({ visible }: Props) {
   );
 
   useEffect(() => {
-    HandlersService.getSudoSosPriceList().then((res) => {
+    getSudoSosPriceList().then((res) => {
```

Comment:
_Informative_: here I think using `then()` is fine (and even necessary), because `useEffect` cannot use `await`.